baseband/vlbi_base/base.py

```diff
@@ -74,6 +74,16 @@ def __init__(self, fh_raw, header0, sample_shape, bps, complex_data,
         self.squeeze = squeeze
 
     @property
+    def squeeze(self):
+        """If `True`, remove any dimensions of length unity from
+        arrays after decoding, and add them when encoding."""
+        return self._squeeze
+
+    @squeeze.setter
+    def squeeze(self, squeeze):
+        self._squeeze = squeeze
```

Comment:
Since we now have a setter, let's do a minimal check with `self._squeeze = bool(squeeze)`